goodtablesio/models/task.py

```diff
@@ -0,0 +1,23 @@
+import logging
+import datetime
+from sqlalchemy.orm import relationship
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy import Column, Unicode, DateTime,  ForeignKey
+from goodtablesio.models.base import Base, BaseModelMixin, make_uuid
+log = logging.getLogger(__name__)
+
+
+class Task(Base, BaseModelMixin):
```

Comment:
Please please let's not call this Task :) Tasks are the celery units of work, 1 Job -> n Tasks (eg `get_validation_conf` + `validate`). Let's call it something different like `InternalJob`, which fits the previous naming nicely (even if 1 `InternalJob` only has one celery task for now).